### PR TITLE
The database was changed

### LINKED ISSUE #12: Replacing database sqlite3 with postgresql
# REQUIREMENTS

- Change the database to postgresql

### PR DESCRIPTION
**Pull request by femm34**
# Description
The database.yml and the Gemfile were modified

- The database was changed, because at the beginning of the project sqlite3 was used

### CODE REVIEW

File: config/database.yml [hunk 1]
```diff
@@ -1,25 +1,19 @@
...
 # Do not set this db to the same as development or production.
 test:
   <<: *default
-  database: db/test.sqlite3
+  database: reloj_test
 
 production:
   <<: *default
-  database: db/production.sqlite3
+  database: reloj_production
```
Comment (YilKanda) [inline config/database.yml:19]: This file should not appear on GitHub, instead there should be an example file: `database.yml.example`

**Review by YilKanda — CHANGES_REQUESTED** [review]

File: config/database.yml (hunk 1 above)
Comment (femm34) [reply config/database.yml:19]: @YilKanda ok thanks, i think that's fixed now 😀

**Review by JuanequeX — APPROVED** [review]
LGTM

**Review by YilKanda — APPROVED** [review]
LGTM!

**Review by xFerchoVJ — APPROVED** [review]
LGFM!🚀

**Pull request merged**